PeriodicTasks.py

```diff
@@ -339,6 +339,31 @@ async def send_marketing_messages(self):
         except Exception as e:
             print(Fore.RED + f"{e} ")
 
+    async def send_builder_ranks(self):
+        stats = self.backoffice.stats_manager.get_top_builders(limit=5)
+        bridges = '\U0001F309'
+        string = ''
+        rank = 1
+        for user in stats:
+            try:
+                username = user['userName']
+                brdiges = user["bridges"]
```

Comment:
There is a typo here "brdiges".
But it's a lucky one, because wihtout it, there would be a variable "collision", with the "bridges" name defined outside of the loop.
You should rename one of the two :)